javascript/.eslintrc

```diff
@@ -110,6 +110,10 @@
         "semi": 2,
 
         // Disallow function definitions of the form 'function myFunc() {'
-        "func-style": [2, "expression"]
+        "func-style": [2, "expression"],
+
+        // Disallow unused variables/expressions
+        "no-unused-vars": 2,
+        "no-unused-expressions": 2
```

Comment:
I would suggest to allow the use of short circuits and ternary conditionals, since I see those a lot.

`"no-unused-expressions": [2, { allowShortCircuit: true, allowTernary: true }]`